packages/react-experiments/src/components/SelectedItemsList/SelectedPeopleList/Items/SelectedPersona.tsx

```diff
@@ -15,6 +15,7 @@ type ISelectedPersonaProps<TPersona> = ISelectedItemProps<TPersona> & {
   isValid?: (item: TPersona) => boolean;
   canExpand?: (item: TPersona) => boolean;
   getExpandedItems?: (item: TPersona) => Promise<TPersona[]>;
+  size?: (item: TPersona) => PersonaSize;
```

Comment:
so I think this is already on the item somewhere, much deeps back in the props. I'll try to find it.

Reply:
Ok yep, it can already be set on the item, so set item.size

Reply:
Thanks! Fixed